src/unity/python/turicreate/toolkits/evaluation.py

```diff
@@ -33,6 +33,19 @@
                                               _check_categorical_option_type
 from turicreate.toolkits._main import ToolkitError as _ToolkitError
 
+try:
```

Comment:
`UserDict` doesn't seem to be getting used anywhere.

Reply:
Thoughts? on the PR Toby?